popup.html

```diff
@@ -8,9 +8,11 @@
     <ul>
         <li>ಠ_ಠ</li>
         <li>(╯°□°）╯︵ ┻━┻</li>
+        <li>┻━┻︵ \(°□°)/ ︵ ┻━┻</li>
```

Comment:
![its-not-going-to-happen-mean-girls](https://cloud.githubusercontent.com/assets/279498/8122223/b10d0e94-106b-11e5-9bdb-c6269c2cccb1.gif)